flare/gp_from_aimd.py

```diff
@@ -189,14 +195,30 @@ def run(self):
             self.pred_func(cur_frame, self.gp)
 
             # Convert to meV/A
-            mae = np.mean(np.abs(cur_frame.forces - dft_forces)) * 1000
+            error = np.abs(cur_frame.forces - dft_forces)
+            mae = np.mean(error) * 1000
+            mae_perspecies = {}
+            count_perspecies = {}
+            species = [Z_to_element(Z) for Z in set(cur_frame.coded_species)]
+            for ele in species:
+                mae_perspecies[ele] = 0
+                count_perspecies[ele] = 0
+            print(list(mae_perspecies.keys()))
+            for atom in range(cur_frame.nat):
+                Z = cur_frame.coded_species[atom]
+                ele = Z_to_element(Z)
+                mae_perspecies[ele] += np.sum(error[atom, :])
+                count_perspecies[ele] += 1
+            for ele in species:
+                mae_perspecies[ele] /= (count_perspecies[ele]*3+1e-25)
+
```

Comment:
Thanks for implementing this! This will be a massive boon to myself and other users.